### PR TITLE
fix fluid type config & font weight

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Fix fluid type configuration
- Font weight 300,400,600 added
- Fluid font was added to explore the page

Task: [APP-540](https://aragonassociation.atlassian.net/browse/APP-540)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/ui-components/tailwind.config.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ module.exports = {
         screenMin: 20, // 20rem === 320px
         screenMax: 96, // 96rem === 1536px
         unit: 'rem',
-        prefix: 'ft',
+        prefix: 'ft-',
```
Comment (WissenIstNacht) [inline packages/ui-components/tailwind.config.js:152]: oh man, what a silly bug...

File: packages/ui-components/tailwind.config.js [hunk 2]
```diff
@@ -164,6 +164,7 @@ module.exports = {
         '3xl': [4, 1.2],
         '4xl': [5, 1.2],
         '5xl': [6, 1.2],
+        '6xl': [7, 1.2],
```
Comment (WissenIstNacht) [inline packages/ui-components/tailwind.config.js]: Are you sure this is needed? The typography only shows up to 5xl. Also you're never actually using this in the code I think.

File: packages/ui-components/src/components/alerts/alertCard.tsx [hunk 3]
```diff
@@ -67,5 +67,5 @@ const Title = styled.div.attrs(({mode}: ContainerProps) => ({
 }))<ContainerProps>``;
 
 const HelpText = styled.div.attrs(() => ({
-  className: 'text-ui-600 text-sm',
+  className: 'text-ui-600 ft-text-sm',
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/alerts/alertCard.tsx:70]: Why only change the helptext but not the title?

File: packages/web-app/index.html [hunk 4]
```diff
@@ -1,78 +1,79 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/src/public/favicon.svg" />
 
-<head>
-  <meta charset="UTF-8" />
-  <link rel="icon" type="image/svg+xml" href="/src/public/favicon.svg" />
+    <!-- TODO: remove once fonts have been downloaded and set in stone -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@300;400;500;600;700&family=Syne:wght@700&display=swap"
```
Comment (WissenIstNacht) [inline packages/web-app/index.html]: Why re-introduce the other font-weights again? Are they now needed somewhere?

File: packages/ui-components/src/components/alerts/alertBanner.tsx [hunk 5]
```diff
@@ -32,5 +32,5 @@ type ContainerProps = {
 };
 
 const Container = styled.div.attrs(({mode}: ContainerProps) => ({
-  className: `text-xs py-0.5 px-2 text-center font-bold ${styles[mode]}`,
+  className: `ft-text-xs py-0.5 px-2 text-center font-bold ${styles[mode]}`,
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/alerts/alertBanner.tsx:35]: Hey, I stumbled upon another issue here regarding the font thinkness (I think I caused this, sorry). Don't worry about it for now, let's discuss in the next daily. I think it's worth opening another bug ticket for it.

**Review by WissenIstNacht — COMMENTED** [review]
Hey Sepehr, thanks a lot for fixing that stop. Must have taken a while to find the issue with the prefix...

I left a couple of comments. They don't necessarily need changing, I'm just curious why you made those changes.

File: packages/ui-components/src/components/alerts/alertBanner.tsx (hunk 5 above)
Comment (Fabricevladimir) [reply packages/ui-components/src/components/alerts/alertBanner.tsx:35]: Yeah something is up with the thickness

**Review by Fabricevladimir — APPROVED** [review]
Thanks, you can merge after addressing the other comments. Small fix generating lots of work 😄

File: packages/ui-components/tailwind.config.js (hunk 2 above)
Comment (sepehr2github) [reply packages/ui-components/tailwind.config.js]: Thanks for mentioning this one! After a quick sync with Selim, I decided to remove this one.

File: packages/ui-components/src/components/alerts/alertCard.tsx (hunk 3 above)
Comment (sepehr2github) [reply packages/ui-components/src/components/alerts/alertCard.tsx:70]: added fluid font base to Title

File: packages/web-app/index.html (hunk 4 above)
Comment (sepehr2github) [reply packages/web-app/index.html]: Yes, According to our discussion with Selim we decided to add for now and later do the refactor for the other parts of the application!

File: packages/ui-components/src/components/alerts/alertBanner.tsx (hunk 5 above)
Comment (sepehr2github) [reply packages/ui-components/src/components/alerts/alertBanner.tsx:35]: We postponed this topic until the MVP review by UX people!

Comment (WissenIstNacht) [reply packages/ui-components/src/components/alerts/alertBanner.tsx:35]: I know what the issue is. Like with this PR, it's just potentially a of small changes. So yeah, let's not think about it right now 👍🏼

File: packages/web-app/index.html (hunk 4 above)
Comment (WissenIstNacht) [reply packages/web-app/index.html]: What do you mean add now and refactor later...? The design system only specifies two font weights, 500 and 700. If the font weights 300, 400 and 600 are not used right now, please remove them.

Comment (WissenIstNacht) [inline packages/web-app/index.html]: So I've checked: The only fonts that are needed are 500 and 700. Please remove the others.

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]

File: packages/web-app/index.html (hunk 4 above)
Comment (sepehr2github) [reply packages/web-app/index.html]: I misunderstood what Selim told me

**Review by WissenIstNacht — APPROVED** [review]

**Pull request merged**